src/components/molecules/search/Search.jsx

```diff
@@ -137,35 +142,34 @@ const Search = ({ autoSuggestLimit = 10 }) => {
 				inputProps={{
 					placeholder: 'Enter keywords or phrases',
 				}}
-				items={autoSuggest.payload || []}
+				items={(!autoSuggest.loading && autoSuggest.payload) || []}
 				getItemValue={(item) => item.termName}
 				shouldItemRender={matchItemToTerm}
 				onChange={(event) => onChangeHandler(event)}
 				onSelect={(value, item) => onSelectHandler(value)}
-				renderMenu={(children) =>
-          (searchText.length >= 3 && autoSuggest.payload )
-          ? (
-						autoSuggest.payload.length > 0 ? (
-							<div
-								className="ncids-autocomplete__menu --terms"
-								role="listbox"
-								data-testid="tid-auto-suggest-options">
-								{children}
-							</div>
+				renderMenu={(children, index) => (
+					<div
+						key={index}
+						className="ncids-autocomplete__menu --terms"
+						role="listbox"
+						data-testid="tid-auto-suggest-options">
+						{searchText.length >= 3 ? (
+							!autoSuggest.loading && autoSuggest.payload?.length ? (
+								children
+							) : (
+								<div className="ncids-autocomplete__menu-item">
+									{autoSuggest.loading
+										? 'Loading results...'
+										: 'No results found'}
```

Comment:
We have removed 'no results found' helper text previously with bug fix - this reintroduces it. Could you replace it with an empty tag

Reply:
Thanks for the catch, missed in rebase. Fixed!